### PR TITLE
OPREF/disable-bottom-up-menu-items

### PR DESCRIPTION
**Pull request by anthony-langford**

### CODE REVIEW

File: src/components/BottomUpMenu/BottomUpMenu.tsx [hunk 1]
```diff
@@ -6,7 +6,8 @@ import Menu from "./Menu";
 export interface Option {
   label: string;
   onPress: () => void;
-  textColor?: string;
+  textColor: string;
```
Comment (ckybryan) [inline src/components/BottomUpMenu/BottomUpMenu.tsx:9]: why making textColor required?

Comment (ckybryan) [reply src/components/BottomUpMenu/BottomUpMenu.tsx:9]: image

If you want to remove it then update docs pls

Comment (anthony-langford) [reply src/components/BottomUpMenu/BottomUpMenu.tsx:9]: Because it's being destructured. Will crash if not provided

Comment (ckybryan) [reply src/components/BottomUpMenu/BottomUpMenu.tsx:9]: can we somehow add a black as a default?

Comment (anthony-langford) [reply src/components/BottomUpMenu/BottomUpMenu.tsx:9]: yes

**Pull request merged**